config/site_settings.yml

```diff
@@ -908,7 +908,9 @@ posting:
     default: true
     client: true
   max_mentions_per_post: 10
-  max_users_notified_per_group_mention: 100
+  max_users_notified_per_group_mention:
+    default: 100
```

Comment:
Should we put a limit on this? This is not designed to mention thousands of people.

Probably a limit of 250 or 500 would already be a lot.

Reply:
Sounds good!